packages/quantic/force-app/examples/main/lwc/exampleInsightPanel/exampleInsightPanel.js

```diff
@@ -0,0 +1,70 @@
+// @ts-ignore
+import youtubeTemplate from './resultTemplates/youtubeResultTemplate.html';
+// @ts-ignore
+import caseTemplate from './resultTemplates/caseResultTemplate.html';
+// @ts-ignore
+import chatterTemplate from './resultTemplates/chatterResultTemplate.html';
+import {LightningElement, api} from 'lwc';
+import {
+  getHeadlessBundle,
+  getHeadlessEnginePromise,
+} from 'c/quanticHeadlessLoader';
+
+export default class ExampleInsightPanel extends LightningElement {
+  /** @type {string} */
+  @api engineId = 'example-insight-panel';
+  /** @type {string} */
+  @api insightId = '6a333202-b1e0-451e-8664-26a1f93c2faf';
+
+  connectedCallback() {
+    this.template.addEventListener(
```

Comment:
It would be nice to remove the event listener when the component is disconnected.